### ISSUE TITLE
Seeking New Maintainer for hyprland-autoname-workspaces

### ISSUE THREAD

**Issue by cyrinux** [hyprland-community/hyprland-autoname-workspaces#117]
Hello everyone,

I’m looking for someone to take over maintenance of the hyprland-autoname-workspaces project. Due to my current time constraints and the fact I can't use hyprland actually, I’m no longer able to keep the project updated and address issues as they come up.

The first key step for moving this project forward is to bump the hyprland-rs library. If you’re interested in helping out, please let me know—your contributions and willingness to maintain would be greatly appreciated!

Thank you in advance for your support and for considering taking on this role. If you have any questions, suggestions, or are interested, feel free to comment here or open a pull request.

**Comment by matt-fff** [comment]
Hey @cyrinux, I wanted to signal my willingness to jump in as a maintainer.  I make heavy use of hyprland-autoname-workspaces and expect that to continue for the foreseeable future. I would hate to see the project stagnate.

I have a [pending contribution to Waybar](https://github.com/Alexays/Waybar/pull/3878) that I put together specifically to fix an issue with how [Waybar interacts with hyprland-autoname-workspaces](https://github.com/Alexays/Waybar/issues/3830).  And I'm still writing tests for a [draft PR](https://github.com/hyprland-community/hyprland-autoname-workspaces/pull/118)  that should improve the way hyprland-autoname-workspaces interacts with Waybar and other similar Hyprland event consumers.

Thanks,
Matt

**Comment by cyrinux** [comment]
I'm glad to read this @matt-fff , thanks, note that you  will  maybe somehow sometimes have to do PR also on https://github.com/hyprland-community/hyprland-rs, and bump it in hyprland-autoname-workspaces ! Maybe @yavko will be happy if you can help there too. I use to help him but I can't too for the same reasons.

I will add you as maintainer.

**Comment by matt-fff** [comment]
Thanks for the heads up about hyprland-rs. I'll get more familiar with it and take a look at upgrading hyprland-autoname-workspaces to the most recent version.

**Comment by mkay-dev** [comment]
Just FYI waybar allows the same functionality out of the box by using the workspace-taskbar sub-module and setting the format to {icon} only.

<img width="218" height="70" alt="Image" src="https://github.com/user-attachments/assets/ea82eead-2892-494d-b77e-0b02344d6dfa" />

**Comment by fiskhest** [comment]
> Just FYI waybar allows the same functionality out of the box by using the workspace-taskbar sub-module and setting the format to {icon} only.
> <img alt="Image" width="218" height="70" src="https://private-user-images.githubusercontent.com/85131358/546749097-ea82eead-2892-494d-b77e-0b02344d6dfa.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzIwODcxNDIsIm5iZiI6MTc3MjA4Njg0MiwicGF0aCI6Ii84NTEzMTM1OC81NDY3NDkwOTctZWE4MmVlYWQtMjg5Mi00OTRkLWI3N2UtMGIwMjM0NGQ2ZGZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMjI2VDA2MjA0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUxZDEzMjRhY2I1MGU3NGNiNWU1NTc3ZDIwMmY1NGFiYjNmZGQzZWIxYjVlMjFmOGE5OTVjODZjNWQyMTNhOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GL71ZkEfx8w_kRqsPzBhSzBpKIIBtO5MJlvAP5z3tQg">

Thanks for the heads up. Had not heard about this before nor found examples of it in the wild.
Here is the documentation: https://github.com/Alexays/Waybar/wiki/Module:-Hyprland#workspace-taskbars-example

**Comment by mkay-dev** [comment]
> Thanks for the heads up. Had not heard about this before nor found examples of it in the wild. Here is the documentation: https://github.com/Alexays/Waybar/wiki/Module:-Hyprland#workspace-taskbars-example

No problem. If someone wants to port this functionality beyond waybar the module code gets the icons from the .desktop file associated with the application you're running via ID (I only gave the code a quick once over so there might be more to it). -Might- be relatively easy to reuse that code:
https://github.com/Alexays/Waybar/blob/7744320ab2c793d092620c2ec859327280b676a6/src/util/icon_loader.cpp#L43
https://github.com/Alexays/Waybar/blob/7744320ab2c793d092620c2ec859327280b676a6/src/AAppIconLabel.cpp

**Comment by shivros** [comment]
Yeah, if you only need the icon, you're better off using native functionality. I use this project to layer on a lot of additional context in my workspaces, beyond the icon.
